Firestore/core/test/firebase/firestore/local/serializer_test.cc

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Firestore/core/src/firebase/firestore/local/serializer.h"
+
+#include "Firestore/Protos/cpp/firestore/local/maybe_document.pb.h"
+#include "Firestore/core/src/firebase/firestore/model/field_value.h"
+#include "Firestore/core/src/firebase/firestore/model/maybe_document.h"
+#include "Firestore/core/src/firebase/firestore/model/snapshot_version.h"
+#include "Firestore/core/src/firebase/firestore/remote/serializer.h"
+#include "Firestore/core/src/firebase/firestore/util/status.h"
+#include "Firestore/core/test/firebase/firestore/testutil/testutil.h"
+#include "google/protobuf/util/message_differencer.h"
+#include "gtest/gtest.h"
+
+namespace local = firebase::firestore::local;
+namespace remote = firebase::firestore::remote;
+namespace v1beta1 = google::firestore::v1beta1;
+
+using firebase::firestore::model::DatabaseId;
+using firebase::firestore::model::Document;
+using firebase::firestore::model::DocumentKey;
+using firebase::firestore::model::MaybeDocument;
+using firebase::firestore::model::SnapshotVersion;
+using firebase::firestore::testutil::Doc;
+using firebase::firestore::util::Status;
+using firebase::firestore::util::StatusOr;
+using google::protobuf::util::MessageDifferencer;
+
+// TODO(rsgowman): This is copied from remote/serializer_tests.cc. Refactor.
+#define EXPECT_OK(status) EXPECT_TRUE(StatusOk(status))
+
+// Fixutre for running the local serializer tests.
```

Comment:
Nit: s/Fixutre/Fixture. Also, I think you can remove the comment altogether, it seems pretty clear what the class does from its base and its code location (optional).

Reply:
Yeah, agreed that it's useless. Removed.